Src/main.c

```diff
@@ -103,7 +111,7 @@ static void MX_USART2_UART_Init(void);
 int main(void)
 {
   /* USER CODE BEGIN 1 */
-
+	uint8_t screenOK = 0;

```

Comment:
Could you comment on what this is used for, please?

Reply:
It specifies whether or not the screen has been detected, I have added a comment clarifying this.